### PR TITLE
Add a Zipkin Virtual Service to route istio zipkin traffic to jaeger

### PR DESCRIPTION
**Pull request by bnookala**

### CODE REVIEW

File: component.yaml [hunk 1]
```diff
@@ -14,4 +16,4 @@ subcomponents:
     method: "git"
   - name: "jaeger"
     source: "https://github.com/bnookala/fabrikate-jaeger"
-    method: "git"
+    method: "git"
```
Comment (evanlouie) [inline component.yaml]: newline please

File: manifests/zipkin-virtualservice.yaml [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+  hosts:
+    - zipkin.jaeger.svc.cluster.local
+  http:
+    - match:
+        - port: 9411
+      route:
+        - destination:
+            host: zipkin.jaeger.svc.cluster.local
+            port:
+              number: 9411
```
Comment (evanlouie) [inline manifests/zipkin-virtualservice.yaml]: newline please

File: component.yaml (hunk 1 above)
Comment (bnookala) [reply component.yaml]: done

File: manifests/zipkin-virtualservice.yaml (hunk 2 above)
Comment (bnookala) [reply manifests/zipkin-virtualservice.yaml]: done

**Pull request merged**